tests/backtest/conftest.py

```diff
@@ -18,12 +22,47 @@ def example_assets() -> dict:
 
 
 @pytest.fixture
-def example_asset_pair() -> AssetPair:
+def asset_pair() -> AssetPair:
     """Example symbol for a given trade."""
     return AssetPair(Ticker("BTC"), Ticker("USDT"))
 
 
 @pytest.fixture
-def example_timestamp() -> datetime.datetime:
+def asset_pair_eth_usdt() -> AssetPair:
+    """Example symbol for a given trade."""
+    return AssetPair(Ticker("ETH"), Ticker("USDT"))
+
+
+@pytest.fixture
+def asset_pair_btc_eth() -> AssetPair:
+    """Example symbol for a given trade."""
+    return AssetPair(Ticker("BTC"), Ticker("ETH"))
+
+
+@pytest.fixture
+def timestamp() -> dt.datetime:
     """Example timestamp for a given trade."""
-    return datetime.datetime.fromtimestamp(int("riddle", 36))
+    return dt.datetime.fromtimestamp(int("riddle", 36))
+
+
+@pytest.fixture
+def timestamp_asset_price(asset_pair: AssetPair) -> dict[AssetPair, pd.DataFrame]:
+    """Example asset price dictionary."""
+    column_names = ["High", "Low", "Open", "Close"]
+    data = [[5.0, 1.0, 2.0, 3.0]]
```

Comment:
Probably doesn't matter, but might be good to add a `volume` column for future compatibility.

Reply:
If it's not being used right now -- going to leave out.